ccf_transaction_processor/generate_signing_keys/key_read.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from infra.clients import CCFClient
+import argparse
+
+def run(args):
+
+    host = args.host
+    port = 6006
+    cert = "./user1_cert.pem"
```

Comment:
(same as above, hardcoded users vs generic util)

Reply:
Thank you, please see above.